rpc/commands/compile.proto

```diff
@@ -34,13 +34,12 @@ message CompileReq {
   bool verbose = 10;     // Turns on verbose mode.
   bool quiet = 11;             // Suppresses almost every output.
   string vidPid = 12;   // VID/PID specific build properties.
-  string exportFile = 13 [deprecated = true]; // DEPRECATED: use exportDir instead
   int32 jobs = 14;   // The max number of concurrent compiler instances to run (as `make -jx`). If jobs is set to 0, it will use the number of available CPUs as the maximum.
   repeated string libraries = 15; // List of custom libraries paths separated by commas.
   bool optimizeForDebug = 16; // Optimize compile output for debug, not for release.
-  bool dryRun = 17; // When set to `true` the compiled binary will not be copied to the export directory.
-  string export_dir = 18; // Optional: save the build artifacts in this directory, the directory must exist.
-  bool clean = 19; // Optional: cleanup the build folder and do not use any previously cached build
+  string export_dir = 17; // Optional: save the build artifacts in this directory, the directory must exist.
```

Comment:
Still not ok :) `export_dir` should be 18, and clean 19, while `export_binaries` becomes 20

Reply:
:facepalm: I'll get to it.